package.json

```diff
@@ -45,7 +46,7 @@
     "react-paginate": "^6.3.2",
     "react-redux": "^7.1.0",
     "react-router-dom": "^5.0.1",
-    "react-scripts": "^3.0.1",
+    "react-scripts": "4.0.3",
```

Comment:
It was possible to upgrade to the 2021 version of the react-scripts.
Further upgrading encountered errors described on the design document and were discarded.

Reply:
I don't think we should update any version in this PR. We have a project to upgrade dependencies and this might break the building of executable, ledger integration, only in one specific OS. If we have plans to merge this PR, it should be only for the test foundation

Reply:
It's a valid point. Reverted back the `react-scripts` to the `^3.0.1` version :+1: